tests/validation_tests/round_four_specific_validations/test_tf_r4_specific_validations.py

```diff
@@ -376,3 +378,29 @@ def test_validate_sign_off_failure(invalid_review_and_sign_off_section):
             message="You must fill out the Date for this section. You need to get sign off from a programme SRO",
         ),
     ]
+
+
+def test_validate_psi_funding_gap():
+    private_funding_sources_df = pd.DataFrame(
+        data=[
+            {
+                "Private Sector Funding Required": 100,
+                "Private Sector Funding Secured": 0,
+                "Additional Comments": pd.NA,
+            },
+        ]
+    )
+    workbook = {"Private Investments": private_funding_sources_df}
+
+    failures = validate_psi_funding_gap(workbook)
+
+    assert failures == [
+        TownsFundRoundFourValidationFailure(
+            tab="PSI",
+            section="Private Sector Investment",
+            message=(
+                'You have entered data with a greater than zero "Private Sector Investment Gap" without providing '
+                "an additional comment. Use the space provided to tell us why"
+            ),
+        )
+    ]
```

Comment:
Could you also add a test for the success situations. Where the gap is not above 0 and where it is but there is a comment.